src/demo/pages/CommandBarPage/examples/data.ts

```diff
@@ -5,24 +5,28 @@ export const items = [
     icon: 'Add',
     ariaLabel: 'New. Use left and right arrow keys to navigate',
     onClick: () => { return; },
+    automationId: 'newItemMenu',
```

Comment:
if you're mixing in props, you should just be able to say `['data-automation-id']: 'foo'` and it should mix it in. Then you can use data-automation-id, or without the dash, or data-foo...